tests/test_core.py

```diff
@@ -45,18 +47,21 @@ def setup_test_table(table_name, sqlalchemy_url):
     test_replication_key_table = Table(
         table_name,
         metadata_obj,
-        Column("id", Integer, primary_key=True),
+        # CrateDB adjustments.
+        Column("id", BigInteger, primary_key=True, server_default=sqlalchemy.text("NOW()::LONG")),
```

Comment:
This emulates PostgreSQL's `SERIAL` type very badly, as it does not feature uniqueness guarantees. However, it is the most low-impact change to make it work ™.

In this case, "make it work" means "satisfying the test suite".